src/Process/CommandExecutor.php

```diff
@@ -30,12 +30,23 @@ public function executeParallel(array $commands, OutputInterface $logOutput, Out
         $manager->mustRun();
     }
 
-    public function passthru(string $command): int
+    public function passthru(string $command, OutputInterface $output): int
     {
         $process = Process::fromShellCommandline($command, null, null, null, null);
-        $process->setTty(true);
-        $process->run();
-        return $process->getExitCode();
+        $process->setTty($output->isDecorated());
+
+        $errorOutput = $output;
+        if ($output instanceof ConsoleOutputInterface) {
+            $errorOutput = $output->getErrorOutput();
+        }
+
+        return $process->run(function ($type, $buffer) use ($output, $errorOutput) {
+            if ($type == \Symfony\Component\Process\Process::OUT) {
```

Comment:
use namespace

Reply:
This shouldn't be your job. And not my Job. Changing Namespaces to `use` Statements. Is there a Tool we could use to enforce this?

Reply:
phpstorm